File: src/main/java/org/oewntk/pojos/ParsePojoException.java
/*
 * Copyright (c) 2021. Bernard Bou.
 */

package org.oewntk.pojos;

/**
 * Exception raised when Pojo object can't be parsed
 */
public class ParsePojoException extends Exception
{
	private static final long serialVersionUID = 1L;

	public ParsePojoException(String message)
	{
		super(message);
	}

	public ParsePojoException(String message, Throwable cause)
	{
		super(message, cause);
	}

	public ParsePojoException(Throwable cause)
	{
		super(cause);
	}
}

File: src/main/java/org/oewntk/pojos/ParsePojoException.kt
/*
 * Copyright (c) 2021-2024. Bernard Bou.
 */
package org.oewntk.pojos

/**
 * Exception raised when Pojo object can't be parsed
 */
class ParsePojoException : Exception {

    constructor(message: String) : super(message)

    constructor(message: String, cause: Throwable) : super(message, cause)

    constructor(cause: Throwable) : super(cause)
}
